### ISSUE TITLE
Update documentation for CA certificate support changes

### ISSUE THREAD

**Issue by tellison** [adoptium/containers#556]
Check and update the "Can I add my internal CA certificates to the truststore?" section of [the official image documentation](https://github.com/docker-library/docs/blob/master/eclipse-temurin/content.md) to reflect [the latest updates](https://github.com/adoptium/containers/pull/538#issue-2256434035) one they are published.

**Comment by tellison** [comment]
@rassie are you ok to take this task?

**Comment by rassie** [comment]
Yeah, sure, I wanted to have prepared a PR for that before the feature PR had been merged, but thought I had a bit more time, it has been accepted with less discussion than I anticipated. Will take a couple of days though, so if someone else would like to a stab before that, they are welcome to do so.

**Comment by tellison** [comment]
> Yeah, sure, I wanted to have prepared a PR for that before the feature PR had been merged, but thought I had a bit more time, it has been accepted with less discussion than I anticipated. Will take a couple of days though, so if someone else would like to a stab before that, they are welcome to do so.

Apologies if it was merged sooner than expected. The PR was not draft and had undergone reviews and approvals. It can be reverted if you think it is not ready.

**Comment by rassie** [comment]
Opened https://github.com/docker-library/docs/pull/2445.
